### PR TITLE
Modal component

### PR DESCRIPTION
**Pull request by coostenbrug**

### CODE REVIEW

**Review by matthewlyle — COMMENTED** [review]
design and docs look good to me 👍

File: components/src/Modal/Modal.js [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+      </Button>
+    ));
+  }
+
+  getModalButtons() {
+    const { buttonAlignment } = this.props;
+
+    if (buttonAlignment === "spaced") {
+      return (
+        <React.Fragment>
```
Comment (clemensp) [inline components/src/Modal/Modal.js]: `<>`

File: components/src/Modal/Modal.js [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+                  {title}
+                </SectionTitle>
+              ) : (
+                <div style={{ height: theme.space.x4 }} />
+              )}
+              {onRequestClose && <ModalCloseButton onClick={onRequestClose} />}
+            </ModalHeader>
+          )}
+          <ModalContent>
+            <InnerModalContent>{children}</InnerModalContent>
```
Comment (clemensp) [inline components/src/Modal/Modal.js]: `SomeComponent` and `InnerSomeComponent` is a design smell.

Why do we need `ModalContent` and `InnerModalContent`?
Is there a way to handle `InnerModalContent` inside `ModalContent`?

File: components/src/Modal/Modal.story.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            <Input name="name" id="name" labelText="Name" />
+            <Input type="number" name="age" id="age" labelText="Age" />
+          </Form>
+        </Modal>
+      </div>
+    );
+  }
+}
+
+storiesOf("Modal", module)
```
Comment (clemensp) [inline components/src/Modal/Modal.story.js:64]: Unused param `module`

File: docs/src/pages/components/navbar.js [hunk 4]
```diff
@@ -14,6 +14,7 @@ import {
   ListItem,
   Text
 } from "@nulogy/components";
+import { theme } from "@nulogy/components/src";
```
Comment (clemensp) [inline docs/src/pages/components/navbar.js]: `theme` is exported by `@nulogy/components`, so you should be able to add it to the list above.

**Review by clemensp — APPROVED** [review]
LGTM.
Left a few comments to consider. :)

File: components/src/Modal/Modal.js (hunk 1 above)
Comment (coostenbrug) [reply components/src/Modal/Modal.js]: Sounds good, Ive changed all instances of React.Fragment to use this syntax to be consistent

File: components/src/Modal/Modal.story.js (hunk 3 above)
Comment (coostenbrug) [reply components/src/Modal/Modal.story.js:64]: module is required for storybook to have hot-refreshing: https://github.com/storybookjs/storybook/issues/2022

**Pull request merged**